capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,67 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class UniversityData {
```

Comment:
nit - add [javadoc for public classes](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)